spec/integration/direct_container_spec.rb

```diff
@@ -147,6 +147,58 @@ class FooHistory < ActiveFedora::Base
         end
       end
     end
+
+    context "deleting members" do
+      before do
+        class FooHistory < ActiveFedora::Base
+           directly_contains :files, is_member_of_relation: ::RDF::URI.new("http://example.com/isWithin")
+        end
+      end
+      after do
+        Object.send(:remove_const, :FooHistory)
+      end
+
+      describe "#delete the contained object directly" do
+        let(:history) { FooHistory.create }
+        let(:file1) { history.files.build }
+        let(:file2) { history.files.build }
+
+        before do
+          file1.content = "hello"
+          file2.content = "hola"
+          history.save
+          history.reload
+        end
+
+        it "deletes the contained resource directly" do
+          expect(history.files).to eq [file1, file2]
+          file1.delete
```

Comment:
@jcoyne could you confirm is this how we want to test the delete in direct containers? (1/2)